website/pages/blog/judging-open-source-by-github-stars.mdx

```diff
@@ -0,0 +1,252 @@
+50th percentile:                     3               E
+55th percentile:                     3               E
+60th percentile:                     3               E
+65th percentile:                     3               E
+70th percentile:                     3               E
+75th percentile:                     3               E
+80th percentile:                     4               E
+85th percentile:                     4               E
+90th percentile:                     4               E
+95th percentile:                     4               E
+----------------------------------------------------------
+Overall trust:                                       E
+```
+
+**Legit Stars (as a control):**
+
+```
+Fetching contributions for 139 users up to year 2013
+Building trust report...ok
+
+Averages                             Score           Trust
+--------                             -----           -----
+Weighted contributions:              118821            A
+Private contributions:               2407              A
+Created issues:                      72                A
+Commits authored:                    1300              A
+Repositories:                        74                A
+Pull requests:                       101               A
+Code reviews:                        69                A
+Account age (days):                  2538              A
+5th percentile:                      145               A
+10th percentile:                     711               A
+15th percentile:                     2726              A
+20th percentile:                     4144              A
+25th percentile:                     8535              A
+30th percentile:                     13650             A
+35th percentile:                     20728             A
+40th percentile:                     23322             A
+45th percentile:                     28244             A
+50th percentile:                     37636             A
+55th percentile:                     42536             A
+60th percentile:                     55776             A
+65th percentile:                     78956             A
+70th percentile:                     93676             A
+75th percentile:                     129164            A
+80th percentile:                     194463            A
+85th percentile:                     231380            A
+90th percentile:                     286798            A
+95th percentile:                     559934            A
+----------------------------------------------------------
+Overall trust:                                         A
+```
+
+So, that's scary. The premium stars scored pretty decently as a `B`, but the cheap ones scored an
+`E` which is the worst, but who knows, maybe there's an ultra premium stars service that's scoring
+an `A`.
+
+## Learned Lessons
+
+Don't judge open source by the number of stargazers, because they can easily and quite affordably be
```

Comment:
```suggestion
Don't judge a book by its cover, or in this context, Don't judge open source by the number of stargazers, because they can easily and quite affordably be
```